CHANGELOG.txt

```diff
@@ -1,3 +1,62 @@
+Version 3.2.0
+====
+
```

Comment:
Maybe we should note that this version is the last to support Python 2.7 (and Python 3.5). We could note that here in the changelog and in the Readme? Maybe we should add a small table in the docs that we link to in the Readme, like:

Our version | Supported Python versions
4.0 | 3.6+ (expected)
3.2 | 2.7, 3.5+
3.1 | 2.7, 3.4+
2.0 | 2.7, 3.2+

(I did not actually look up the versions, just as an example)

Reply:
Good idea - done.

Reply:
Nice! :)